packages/ckeditor5-dev-env/lib/release-tools/tasks/bump-year.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const fs = require( 'fs' );
+const glob = require( 'glob' );
+const minimatch = require( 'minimatch' );
+
+/**
+ * Updates year in all licenses in the provided directory, based on provided glob patterns.
+ *
+ * Replaces:
+ * 	Copyright (c) [initial year]-xxxx
+ *
+ * With:
+ *	Copyright (c) [initial year]-[current year]
+ *
+ * @param {Object} params
+ * @param {String} params.cwd Current working directory from which all paths will be resolved.
+ * @param {Number} params.initialYear Year from which the licenses should begin. Default value is 2003.
+ * @param {Array} params.globPatterns array of objects, where each object has string property 'pattern',
+ * and optionally 'options' property.
+ */
+module.exports = function bumpYear( params ) {
+	if ( !params.initialYear ) {
+		params.initialYear = '2003';
+	}
+
+	const filesToUpdate = params.globPatterns
+		.map( globPattern => glob.sync( globPattern.pattern, globPattern.options ) )
+		.reduce( ( previous, current ) => [ ...previous, ...current ] )
+		.filter( fileName => {
+			// Filter out directories.
+			if ( fs.statSync( fileName ).isDirectory() ) {
+				return false;
+			}
+
+			// Filter out nested `node_modules`.
+			if ( minimatch( fileName, '**/node_modules/**' ) ) {
+				return false;
+			}
+
+			// Filter out builds.
+			if ( minimatch( fileName, '**/build/**' ) ) {
+				return false;
+			}
+
+			return true;
+		} );
+
+	const filesWithoutHeader = [];
+	const totalFiles = filesToUpdate.length;
+	let updatedFiles = 0;
+
+	filesToUpdate.forEach( fileName => {
+		const fileContent = fs.readFileSync( fileName, 'utf-8' );
+		const currentYear = new Date().getFullYear();
+
+		// This syntax has to be used in order to insert variable into the regex.
+		const yearRegexp = new RegExp( `(?<=Copyright \\(c\\) ${ params.initialYear }-)\\d{4}`, 'gm' );
+
+		let updatedFileContent = fileContent.replace( yearRegexp, currentYear );
+
+		// TODO: Remove this line after all repositories are updated.
+		updatedFileContent = updatedFileContent.replace( /- Frederico Knabben/, 'Holding sp. z o.o' );
```

Comment:
This does not replace all old sentences.

Because of that, my proposal is to focus on the mandatory scope (changing the years) and do not change the license.